packages/jaspr/lib/src/components/html/text.dart

```diff
@@ -47,7 +50,14 @@ Component a(List<Component> children,
       if (type != null) 'type': type,
       if (referrerPolicy != null) 'referrerpolicy': referrerPolicy.value,
     },
-    events: events,
+    events: {
```

Comment:
This file is auto-generated by running 'melos run generate:html'.
Script is at 'packages/jaspr/tool/generate_html.dart' and data source '.../tool/data/html.json'.
So we cannot just modify it, but need to make a solution in the script.

Reply:
@slightfoot I pushed a change, lmk what you think.

Reply:
I see the change, that works. reverting these changes